packages/mobile-top-nav/src/data/menus.js

```diff
@@ -0,0 +1,337 @@
+const texts = {
```

Comment:
unsure how we want to pass through review app URLs as 🐢 s all the way down, just noting again as I pass through the code

Reply:
We can pass in properties to our components to solve this. For instance in the demo I made, we passed in app configuration info into it. ie:
```
// index.html
const appConfig = { 'baseUrl': 'archive.org' }
<ia-app .config=appConfig></ia-app>
```
You can take the same approach here since this component needs environmental information. Pass it in as a configuration object.

Reply:
I'm also inclined to keep as-is for now and update to use the `baseUrl` when the work for full responsive nav happens